### ISSUE TITLE
About the running log

### ISSUE THREAD

**Issue by XuRui314** [kohjingyu/gill#27]
Thanks for sharing this excellent work! Since many CC3M images have expired, I used the LLava-595k CC3M subset for training. Can you provide the training log based on complete CC3M, thx.

**Comment by XuRui314** [comment]
The training results under the CC3M subset perform poorly. Since the paper does not show the metrics used for training. Could you please provide me with a training log based on the complete CC3M to give me some guidance. Thanks.

**Comment by kohjingyu** [comment]
Sorry, I don't think I have the training log at the moment. What is the performance like when training on the 595k subset? I think CC3M should still have > 2M images remaining now, right? Perhaps training on those might could you a better signal?

**Comment by XuRui314** [comment]
The 595k CC3M subset's validation metrics such as captioning acc converged to 34% after 20k steps with batch size128, which i think is very low. Now i'm using CC15m + sbu dataset from LAVIS for trying. Thanks for replying :)

**Comment by XuRui314** [comment]
Sorry to bother you again😂, but I did have some problems reproducing the performance of the paper:
1. I want to evaluate the pre-trained model in dir checkpoints/ , but its weights seem to be pruned, so I can't use it directly
2. I tried batch size 200 on CC15m, training with 4 A100s for 20k steps, all settings are the same as the paper, but the metric values are as low as LLava595k. So i wonder if you can provide some guidance. Thank you
